### ISSUE TITLE
Running `ember g component-test` installs ember-cli-htmlbars-inline-precompile

### ISSUE THREAD

**Issue by mydea** [emberjs/ember.js#18490]
Running `ember g component-test XXX` will install (a very old and outdated) version of ember-cli-htmlbars-inline-precompile:

```ember g component-test f-radio-buttons
installing component-test
  create tests/integration/components/f-radio-buttons/component-test.js
  install package ember-cli-htmlbars-inline-precompile

🚧  Installing packages... This might take a couple of minutes.
Yarn: Installed ember-cli-htmlbars-inline-precompile@^0.3.1
```

Even though with ember-cli-htmlbars >= 4, you don't need to install that package anymore separately. 

---

Output from `ember version --verbose && npm --version && yarn --version`:
```
ember-cli: 3.13.1
http_parser: 2.8.0
node: 10.16.3
v8: 6.8.275.32-node.54
uv: 1.28.0
zlib: 1.2.11
brotli: 1.0.7
ares: 1.15.0
modules: 64
nghttp2: 1.39.2
napi: 4
openssl: 1.1.1c
icu: 64.2
unicode: 12.1
cldr: 35.1
tz: 2019a
os: linux x64
6.9.0
1.19.0
```

(Superseding https://github.com/ember-cli/ember-cli/issues/8886)

**Comment by rwjblue** [comment]
Should have been fixed by https://github.com/emberjs/ember.js/pull/18410, which is included in 3.14.0-beta.1 and higher. What `ember-source` version are you running?

**Comment by mydea** [comment]
3.13.x, so I guess that'll be fine then ;) Thanks, I'll close this then!
